benchmark/benchmarking-multiplexed-session.js

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright 2025 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const muxEnabledTrue = [];
+const muxEnabledFalse = [];
+const transaction_times = [];
+async function main(
+  instanceId,
+  databaseId,
+  projectId,
+  multiplexedEnabled,
+  numThreads,
+  numQueries
+) {
+  async function readQuery(database) {
+    const startTime = Date.now();
+    const query = {
+      sql: 'SELECT * FROM Singers',
+    };
+    await database.run(query);
+    const operationTime = Date.now() - startTime;
+    transaction_times.push(operationTime);
+  }
+
+  async function runQueries() {
+    multiplexedEnabled === 'true'
```

Comment:
Every thread is creating new Spanner instance, that would mean it will create a new mux session. So eventually we are running only one query at a time for one spanner client.